frontend/app/src/components/price-manager/historic/HistoricPriceTable.vue

```diff
@@ -64,11 +62,17 @@ const headers = computed<DataTableHeader[]>(() => [
   }
 ]);
 
+const edit = (item: HistoricalPrice) => {
+  emit('edit', item);
+};
+const refresh = (modified: boolean | HistoricalPrice = false) =>
+  emit('refresh', modified);
+
 const deletePrice = async (item: HistoricalPrice) => {
```

Comment:
Since all the other actions are emitting events and are handled to the parent component wouldn't it make sense to also have this moved to?

Reply:
Hmm, I'm not sure I follow, this only call the emit, the refresh happens on parent component🧐

Reply:
it was about doing `emit('delete', item)` and moving the delete call to the parent too, since now it is the only call to API in this component. Just to have them in proximity.

Is there something that stops us from this? Do you think there is a reason why we should not do this?

Reply:
oh damn I was looking at the wrong line, I thought you highlight the refresh function, okay2, I got it